src/main/java/roomescape/service/ReservationService.java

```diff
@@ -0,0 +1,59 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import roomescape.domain.Reservation;
+import roomescape.dto.RequestReservationDto;
+import roomescape.global.CustomException;
+import roomescape.global.ErrorCode;
+import roomescape.repository.ReservationRepository;
+import roomescape.repository.TimeRepository;
+
+@Service
+@Transactional(readOnly = true)
+public class ReservationService {
+
+    private final ReservationRepository reservationRepository;
+    private final TimeRepository timeRepository;
+
+    public ReservationService(final ReservationRepository reservationRepository, final TimeRepository timeRepository) {
+        this.reservationRepository = reservationRepository;
+        this.timeRepository = timeRepository;
+    }
+
+    @Transactional
+    public Reservation createReservation(RequestReservationDto reservationDto) {
+
+        Map<String, String> reservations = new HashMap<>();
+        reservations.put("name", reservationDto.getName());
+        reservations.put("date", reservationDto.getDate());
+        reservations.put("time_id", reservationDto.getTime());
+
+        Reservation reservation = reservationRepository.insert(reservations);
+
+        return reservation;
+    }
```

Comment:
```suggestion
    @Transactional
    public Reservation createReservation(RequestReservationDto reservationDto) {

        // 추가로 아래의 reservations 변수명이 좀 어색한 것 같아요!
        Map<String, String> reservations = new HashMap<>();
        reservations.put("name", reservationDto.getName());
        reservations.put("date", reservationDto.getDate());
        reservations.put("time_id", reservationDto.getTime());

        return reservationRepository.insert(reservations);
    }
```
- 의미 전달을 크게 해치지도 않고 해당 메서드에서 사용할 객체가 아니기 때문에 변수에 할당하지 않아도 괜찮을 것 같은데 상진님은 어떻게 생각하시나요?

Reply:
> * 의미 전달을 크게 해치지도 않고 해당 메서드에서 사용할 객체가 아니기 때문에 변수에 할당하지 않아도 괜찮을 것 같은데 상진님은 어떻게 생각하시나요?

이 부분에 있어 고민을 했습니다. 바로 return을 할지 객체를 만들어서 반환을 할지에 대해 생각했을 때 객체를 할당했다고 리소스에  큰 피해가 없고 가독성이 좋다고 느껴져 사용했습니다.